pkgs/top-level/all-packages.nix

```diff
@@ -32448,7 +32448,7 @@ with pkgs;
 
   kid3 = libsForQt5.callPackage ../applications/audio/kid3 { };
 
-  kile = libsForQt5.callPackage ../applications/editors/kile { };
+  kile = qt6Packages.callPackage ../applications/editors/kile { };
```

Comment:
Looks like `qt6Packages.callPackage` isn't unnecessary here, since you've been using `qt6.qtbase` instead of `qtbase`, and that could also apply to `qt6.wrapQtAppsHook`?